README.md

```diff
@@ -213,11 +233,77 @@ To regenerate agent source, scraping the pre-existing source to avoid clearing o
+| opensearch             |                                       | 9200, 9600                   | OPENSEARCH_PORT1, OPENSEARCH_PORT2 |                                                                      |
+| opensearch-dashboards  |                                       | 5601                         | OPENSEARCH_DASH_PORT               |                                                                      |
+| postgres               | Postgres SQL Database                 | 5432                         | DB_PORT                            |                                                                      |
+| adminer                | DB Management Web Tool                | http://localhost:8080, 8080  | ADMINER_PORT                       |                                                                      |
+| mqtt-broker            |                                       | 1883                         | MQTT_PORT                          |                                                                      |
+| grafana                |                                       | http://localhost:3000, 3000  | GRAFANA_PORT                       |                                                                      |
+| grafana-image-renderer |                                       | 8081                         | RENDERER_PORT                      |                                                                      |
+| redis                  |                                       | 6379                         | REDIS_PORT                         |                                                                      |
+| anms-ui                |                                       | http://localhost:9030, 9030  | ANMS_UI_HTTP_PORT                  |                                                                      |
+| anms-ui                |                                       | https://localhost:9443, 9443 | ANMS_UI_HTTPS_PORT                 |                                                                      |
+| anms-core              |                                       | 5555                         | ANMS_CORE_HTTP_PORT                |                                                                      |
+| ion-manager            | ION DTN Network Manager (NM) REST API | http://localhost:8089, 8089  | ION_MGR_PORT                       |                                                                      |
+| ion-manager            | DTN Bundle Protocol                   | 4556                         | ION_BP_PORT                        |                                                                      |
+| ion-manager            | Licklider Transmission Protocol (LTP) | 1113                         | ION_LTP_PORT                       |                                                                      |
+
+## Troubleshooting
```

Comment:
Troubleshooting belongs in the Product Guide, and these might belong in a separate part for (non-production deployment) development troubleshooting.